CHANGELOG.md

```diff
@@ -0,0 +1,110 @@
+# Changelog
+
+
+
+## [Unreleased]
```

Comment:
Do we need this **[Unreleased]** section at all? If we are incorporating the changelog updates to the Pull Requests workflow like I suggested, this is a bad addition. It would only be useful if changes to this document are merged at the start of the work for each new feature, which is hardly manageable. Instead, my suggestion is to add the new feature/fix/etc to the changelog **as if it was already merged** along with the respective PR. That way, it will be merged at the same time, and only then show up in the project. Does that make sense? I think it's the most productive way to manage this document without overloading any single contributor.

Also, it this is the case, this PR should include the additions to the CONTRIBUTING.md file, as my version did. Something along the lines of [this commit](https://github.com/inuitcss/inuitcss/pull/229/commits/8a186aecb99bced49317a9e95a37cf2567a7410a).

Reply:
> **Delineate your changes in `CHANGELOG.md`.** Under “[Unreleased]”, make an 
   entry in the the appropriate section, i.e. “Breaking changes”, “Fixes”,
   “New features” or “Minor changes”.

Reply:
I think, it's a good idea to always provide the changelog information alongside each Pull Request. So, apart from the code work done for the feature itself, the changelog entry for that feature should be part of the PR, as well. 

Since we don't know for sure which version this feature will end up being released for, we collect all these changes under “[Unreleased]” and group all changes further in respective categories (i.e. “Breaking changes”, “Fixes”, “New features” or “Minor changes”). When it comes to releasing a new version, we change “[Unreleased]” to the actual version number (directly on develop) and then release the new version.